src/api-service/__app__/onefuzzlib/orm.py

```diff
@@ -85,6 +86,11 @@ def process_state_update(obj: HasState) -> None:
     func = getattr(obj, obj.state.name, None)
     if func is None:
         return
+
+    get_keys = getattr(obj, "get_keys", None)
+    if get_keys is not None:
+        logging.info("processing state update: %s - %s", get_keys(), obj.state.name)
```

Comment:
This is just for extra debuggability, right?

Reply:
Yep.